bin/ajax

```diff
@@ -442,6 +532,23 @@ def delete_data(rd, path, data_type, test=True):
         raise ValueError(f"Something went wrong we wanted to delete {path}!")
 
 
+def check_rundoc_for_live_and_raw(rd):
+    """
+    Count the number of files of live_data (cannot be >1) and raw_data (transfers can get
+    it greater than 1)
+    :param rd: rundoc
+    :return: length 2 tuple with n_live_data and n_raw_records being the number of files
+    in the rundoc for the live data and raw records respectively.
+    """
+    n_live_data, n_raw_records = 0, 0
+    for dd in rd['data']:
+        if dd['type'] == 'live' and dd['location'] != 'deleted':

```

Comment:
The second half of this boolean shouldn't do anything, once we delete data it isn't in `rd['data']` anymore.

Reply:
True, it's some legacy code. I just queried and I think we indeed have no more `{"data.location":"deleted"}` in the runsdb

Reply:
Removed in 983958e